tests/structuredheadings/autonumber.js

```diff
@@ -1,114 +0,0 @@
-/* bender-tags: editor,unit */
```

Comment:
Could you explain why this file is deleted? I understand that these are only tests.